packages/terra-list/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * Changed
   * Sections and subsections updated to add programmatic context to communicate hierarchal relationships
 
+* Added
+  * Prop `instructions` - Conveys information about the list and how to interact with it.
```

Comment:
Can we also add the `instructionText` prop here as well

Reply:
[a19a2da](https://github.com/cerner/terra-core/pull/3688/commits/a19a2da3282722c18081b5625394f9642cc3eaae)